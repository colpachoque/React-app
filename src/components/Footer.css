.footer {
    display: flex;
    width: 100%;
    height: 50px;

    background-color: #b27bdb;
    color: white;
}