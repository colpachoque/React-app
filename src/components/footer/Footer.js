import React from 'react';
import styles from './Footer.css';
import Copyright from '../common/Copyright';

const Footer = (props) => (
            <footer className={styles.footer}>
                <Copyright/>
            </footer>
        );

export default Footer;