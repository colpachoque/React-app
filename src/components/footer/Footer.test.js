import React from "react";
import Footer from "./Footer";
import { shallow, mount } from "enzyme";
import toJson from "enzyme-to-json";

describe("Footer component", () => {
  it("should be render correctly", () => {
    const component = shallow(<Footer />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
