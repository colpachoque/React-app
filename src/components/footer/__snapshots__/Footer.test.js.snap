// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should be render correctly 1`] = `
<footer
  className="footer"
>
  <Copyright />
</footer>
`;
